project/Settings.scala

```diff
@@ -138,12 +198,16 @@ object Settings extends Build {
     }
   )
 
-  /* By default, assembly is not enabled for the demos module, but it can be enabled with
-    `-Ddemos.assembly=true`. From the command line this would be:
-     sbt -Ddemos.assembly=true twitter/assembly */
-  lazy val demoAssemblySettings =
-    if (System.getProperty("demos.assembly", "true").toBoolean) sbtAssemblySettings
-    else Seq.empty
+  lazy val theme = PromptTheme(List(
```

Comment:
Hrm, did not even consider that obviously. I'll see if the plugin can handle that even.